app/views/partials/attachments_edit_list.scala.html

```diff
@@ -48,7 +48,9 @@
                                type="checkbox"
                                value="@file.id"
                                aria-label="publish @file.fileName on GOV.UK"
-                               @if(file.public) {checked="checked"}/>
+                                @if(file.public){
+                                  checked="checked"/>
+                        }
```

Comment:
Should the `/>` here be outside of the curly brace like it was previously? I think otherwise the `input` tag will be unclosed

Reply:
I think the checkbox here should now be controlled by `shouldPublishToRulings`, and perhaps this input should also be disabled if `!file.public`?

Reply:
As discussed, we have now change the logic so that the box is disabled.